libnetwork/resolver_unix.go

```diff
@@ -16,14 +18,22 @@ const (
 	postroutingChain = "DOCKER_POSTROUTING"
 )
 
-func (r *Resolver) setupIPTable() error {
+func (r *Resolver) setupNAT() error {
```

Comment:
Since you're already touching this function signature and all call sites, it seems like an ideal opportunity to do a touch of context plumbing.
```suggestion
func (r *Resolver) setupNAT(ctx context.Context) error {
```

Reply:
This unexported method is only called from one (exported) method, which has no context. So it's a bit futile but, done.